bot/exts/evergreen/emojis.py

```diff
@@ -0,0 +1,39 @@
+import textwrap
+
+from discord import Color, Embed, Emoji
+from discord.ext import commands
+
+from bot.utils.time import time_since
+
+
+class Emojis(commands.Cog):
+    """Has commands related to emojis."""
+
+    @commands.group(name="emoji", invoke_without_command=True)
+    async def emojis_group(self, ctx: commands.Context, emoji: Emoji) -> None:
+        """A group of commands related to emojis."""
+        # No parameters = same as invoking info subcommand
+        await ctx.invoke(self.info_command, emoji)
+
+    @emojis_group.command(name="info")
+    async def info_command(self, ctx: commands.Context, emoji: Emoji) -> None:
+        """Returns relevant information about a Discord Emoji."""
+        emoji_information = Embed(
+            title=f'Information about "{emoji.name}"',
+            description=textwrap.dedent(f"""
+                Name: {emoji.name}
+                Created: {time_since(emoji.created_at)}
+                ID: {emoji.id}
+                Emote source: [Link]({emoji.url})
+            """),
+            color=Color.blurple()
+        )
+        emoji_information.set_thumbnail(
+            url=str(emoji.url)
```

Comment:
You can actually just give it `emoji.url` as set_thumbnail will internally call str before using it. Which would shorten it enough to one line it

Reply:
No, it won't work. `emoji.url` is `discord.Asset` and needs to be explicitly casted to `str`.

Reply:
ah actually wait

Reply:
Which, as i said, set_thumbnail does internally. https://github.com/Rapptz/discord.py/blob/master/discord/embeds.py#L337

Reply:
Yeah yeah, I'm sorry. I had problems before when setting `title` to something other than `str` before and I thought here it would happen the same.